tests/testMatchFits.py

```diff
@@ -44,20 +44,20 @@ def setUp(self):
         self.cat1 = afwTable.SimpleCatalog(self.schema)
         self.cat2 = afwTable.SimpleCatalog(self.schema)
         for i in range(self.size):
-            record1 = self.cat1.table.makeRecord()
-            record2 = self.cat2.table.makeRecord()
+            record1 = self.cat1.makeRecord()
+            record2 = self.cat2.makeRecord()
```

Comment:
In general I'm not sure it's a good idea to effectively change the API in a test. Is there a reason this method was moved from the table to the catalog?